### PR TITLE
Title Fix and minor login fix

### PR DESCRIPTION
**Pull request by jonnykry**
# Overview

This PR fixes the Title SVG's to be clickable (https://github.com/jonnykry/expresso/issues/58).  They were before, but the link was a bit lower than the actual SVG.  After investigation, it seemed to be because we were using `object` -- I switched to `img` and it works fine (source: https://stackoverflow.com/questions/11374059/make-an-html-svg-object-also-a-clickable-link).

Additional quick fixes:
- added `Link` to `/` inside `Title` since all instances of Title were linking to home
- fixed a bug with `this.props.location.state` being undefined
- already authenticated users that access the login page are sent to dashboard

### CODE REVIEW

File: src/components/LoginContainer.js [hunk 1]
```diff
@@ -27,26 +37,17 @@ class LoginContainer extends Component {
...
-            this.email = i;
+        return (ref => {
+            this.email = ref;
         });
     }
 
     _password() {
-        return (i => {
-            this.password = i;
+        return (ref => {
```
Comment (ghmeier) [inline src/components/LoginContainer.js:46]: i mean.....

**Comment by ghmeier** [comment]
LGTM :+1:

File: src/components/LoginContainer.js (hunk 1 above)
Comment (jonnykry) [reply src/components/LoginContainer.js:46]: I had no clue what was happening lol, didn't know what `i` was and thought I'd change it for REFerence 😅

Comment (jonnykry) [reply src/components/LoginContainer.js:46]: .... did you see what I did there?

Comment (jonnykry) [reply src/components/LoginContainer.js:46]: REFerence..?

Comment (jonnykry) [reply src/components/LoginContainer.js:46]: I'll see myself out.

Comment (ghmeier) [reply src/components/LoginContainer.js:46]: I mean, `i` stands for input sooooo.... https://facebook.github.io/react/docs/refs-and-the-dom.html

Comment (ghmeier) [reply src/components/LoginContainer.js:46]: I also used `i` in every other place where we had string refs before, so this is the outlier now

**Pull request merged**